### PR TITLE
Terminal arrays

### PR DESCRIPTION
**Pull request by AloeareV**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -386,9 +425,30 @@ fn annotate_array(result_chars: &mut std::str::Chars) -> serde_json::Value {
             _ => panic!("character is UTF-8 but not ASCII!"),
         }
     }
+
     Value::Array(ordered_results)
 }
 
+fn get_arrray_terminal(viewed: String) -> Value {
```
Comment (zancas) [inline src/lib.rs:432]: Three?

Comment (AloeareV) [reply src/lib.rs:432]: :pirate_flag:

File: src/lib.rs [hunk 2]
```diff
@@ -386,9 +425,30 @@ fn annotate_array(result_chars: &mut std::str::Chars) -> serde_json::Value {
...
+    let viewed_lines = viewed
+        .trim_end()
+        .lines()
+        .map(|line| line.to_string())
+        .collect::<Vec<String>>();
+    dbg!(&viewed_lines);
+
+    let raw_label = if viewed_lines.len() == 1 {
```
Comment (zancas) [inline src/lib.rs:440]: Hmmm...

File: src/lib.rs [hunk 3]
```diff
@@ -386,9 +425,30 @@ fn annotate_array(result_chars: &mut std::str::Chars) -> serde_json::Value {
...
+
     Value::Array(ordered_results)
 }
 
+fn get_arrray_terminal(viewed: String) -> Value {
+    let viewed_lines = viewed
+        .trim_end()
+        .lines()
+        .map(|line| line.to_string())
+        .collect::<Vec<String>>();
```
Comment (zancas) [inline src/lib.rs:437]: Helper?

File: src/lib.rs [hunk 4]
```diff
@@ -386,9 +425,30 @@ fn annotate_array(result_chars: &mut std::str::Chars) -> serde_json::Value {
...
+    dbg!(&viewed_lines);
+
+    let raw_label = if viewed_lines.len() == 1 {
+        &viewed_lines[0]
+    } else {
+        &viewed_lines[1]
+    }
+    .split(|c| c == '(' || c == ')')
```
Comment (zancas) [inline src/lib.rs:445]: Helper?

**Pull request merged**